protocol/avdl/keybase1/phone_numbers.avdl

```diff
@@ -10,13 +10,30 @@ protocol phoneNumbers {
       PhoneNumber phoneNumber;
       @jsonkey("verified")
       boolean verified;
+      @jsonkey("visibility")
+      IdentityVisibility visibility;
       @jsonkey("ctime")
       UnixTime ctime;
+
+    }
+    record PhoneNumberLookupResult {
+       @jsonkey("phone_number")
+       PhoneNumber phoneNumber;
+       @jsonkey("coerced_phone_number")
+       PhoneNumber coercedPhoneNumber;
```

Comment:
so `phoneNumber` and `coercedPhoneNumber` are the same type? I thought that type will mean "well-formed, E164 phone number with proper region code".

Reply:
yeah maybe weird...but the idea is client doesn't do any input validation and treats these as string wrappers. but since this is the only case where we expected a malformed number I could add a RawPhoneNumber type for this endpoint?

Reply:
thx